src/css/_app.scss

```diff
@@ -181,13 +181,25 @@
         .input-group input {
             background-color: map-get($theme-colors, "primary-background-input");
         }
+        a {
+            color: #ffffff;
+        }
     }
   
 
     .welcome-col-right {
-        -ms-flex: 0 0 58.333333%;
-        flex: 0 0 58.333333%;
-        max-width: 58.333333%;
+        -ms-flex: 0 0 50%;
+        flex: 0 0 50%;
+        max-width: 50%;
+        .info-paragraph {
+            margin-bottom: 0 !important;
+        }
+        .info-section-text {
+            margin-top: 0.5rem
+        }
+        a {
+            color: #ffffff;
+        }
```

Comment:
Bump `a` up a level rather than repeating?

Reply:
That did not seem to work, but that could be my inexperience with `scss`.  Going to fix the test and if I don't hear otherwise, merge.  Thanks for the suggestion, though and if you know what's up with that, let me know.